app/workers/rapid_ready_for_decision/form526_asthma_job.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'lighthouse/veterans_health/client'
+
+module RapidReadyForDecision
+  class Form526AsthmaJob < Form526BaseJob
+    STATSD_KEY_PREFIX = 'worker.fast_track.form526_asthma_job'
+
+    sidekiq_options retry: 2
+
+    def assess_data(form526_submission)
+      client = lighthouse_client(form526_submission)
+      client.list_resource('medication_requests')
+
+      ActionMailer::Base.mail(
+        from: ApplicationMailer.default[:from],
+        to: Settings.rrd.event_tracking.recipients,
+        subject: 'Asthma RRD Claim - Offramped',
```

Comment:
For consistency with [recent PR](https://github.com/department-of-veterans-affairs/vets-api/pull/9349/files#diff-ed1e5d9b851dfdcbdc20273d00ded834be82a60b2df94d4dcc24482c858ac968R11)
```suggestion
        subject: 'RRD claim - Offramped',
```